orc8r/cloud/go/services/certifier/servicers/certifier.go

```diff
@@ -692,12 +694,12 @@ func (srv *CertifierServer) getPolicyFromToken(token string) (*certprotos.Policy
 	return policy, nil
 }
 
-func (srv *CertifierServer) getPolicyDecisionFromTokenMany(tokens *certprotos.TokenList, getPDReq *certprotos.GetPolicyDecisionRequest) (*certprotos.GetPolicyDecisionResponse, error) {
+func (srv *CertifierServer) getPolicyDecisionFromTokenMany(ctx context.Context, tokens *certprotos.TokenList, getPDReq *certprotos.GetPolicyDecisionRequest) (*certprotos.GetPolicyDecisionResponse, error) {
```

Comment:
Do we still need this? I think initially when I build it out, I was under the impression that a single user will have multiple tokens, though so far it's just one